### ISSUE TITLE
Interpret latent space

### ISSUE THREAD

**Issue by systats** [fdavidcl/ruta#32]
Hi - in general, I'd like to use your package for topicmodeling on large scale text collections. I managed to get document level vectors with this code. 

```
network <-
  input() +
  dense(256, "elu") +
  variational_block(4, seed = 42, epsilon_std = .5) +
  dense(256, "elu") +
  output("sigmoid")

learner <- autoencoder_variational(network, loss = "binary_crossentropy")
model <- learner %>% train(x_train_mat, epochs = 5)

# summary(learner$network)

inputs <- get_layer(model$models[[1]], index = 1)$input
latent_space <- get_layer(model$models[[1]], index = 7)$output

latent_model <- keras_model(
  inputs = inputs,
  outputs = latent_space
)
```
Via PCA the output looks like this: 

![image](https://user-images.githubusercontent.com/22753281/53825619-0049b780-3f77-11e9-87ec-f7f2fb66b08a.png)

Now I have two questions:

1. Do you know whether there is a better way of  interpreting the latent space? LDA outputs topic proportions. I thought maybe a sigmoid or (softmax for sparsity) activation at the bottleneck. But this way the model does not learn anything useful.

2. Do you know how I could get word-level features that are associated with the each latent dimension?  

Thanks in advance!

Best Simon 

(PS: if you prefer a fully reproducible example, please let me know).

**Comment by fdavidcl** [comment]
Hi,

Thanks for your questions. In general, latent models should be accessible from `model$models$encoder` but the package seems to miscalculate the corresponding layer in this case, since there are several layers included in the "variational_block". I'll file a separate issue for this matter.

Regarding your questions:

1. As far as I know, there is no "meaning" to features learned by a standard AE. The latent features of a basic variational AE do not necessarily represent anything useful either, apart from the probability-theoretical sense ([this paper](https://arxiv.org/abs/1703.01488) seems related but I haven't had the time to look at it closely). An adversarial AE may learn interesting variables if an appropriate prior is chosen but I think you have to discover the meaning after training, by exploring/sampling.
2. Since I am not very used to text-oriented models, I am afraid I currently do not know that.
